ArduPlane/quadplane.cpp

```diff
@@ -3583,4 +3593,14 @@ bool QuadPlane::show_vtol_view() const
     return show_vtol;
 }
 
+// return the PILOT_VELZ_MAX_DN value if non zero, otherwise returns the PILOT_VELZ_MAX value.
+uint16_t QuadPlane::get_pilot_velocity_z_max_dn() const
+{
+    if (pilot_velocity_z_max_dn == 0) {
+        return abs(pilot_velocity_z_max_up);
+    } else {
+        return abs(pilot_velocity_z_max_dn);
+    }
```

Comment:
```suggestion
    return abs(pilot_velocity_z_max_dn);
```

Reply:
this suggested change breaks the code....had to add back the brace